crates/aptos/src/config/mod.rs

```diff
@@ -150,6 +193,91 @@ impl CliCommand<BTreeMap<String, ProfileSummary>> for ShowProfiles {
     }
 }
 
+/// Delete the specified profile.
+#[derive(Parser, Debug)]
+pub struct DeleteProfile {
+    /// Which profile to delete
+    #[clap(long)]
+    profile: String,
+}
+
+#[async_trait]
+impl CliCommand<String> for DeleteProfile {
+    fn command_name(&self) -> &'static str {
+        "DeleteProfile"
```

Comment:
Are these must haves? Currently users can directly modify the config file for deletion and renaming

Reply:
The requirement to manually modify a config file is a clunky pain point repeated across the docs site, and it introduces possiblity for error especially when people are copy-pasting across shell terminals

These changes remove the requirement that anyone need ever manually modify their config, which is especially important when opportunities for mismatches are high enough in advanced operations like auth key rotation where the address of one account might equal the auth key of another account